### PR TITLE
Move: StefanKueng.StExBar version 11.11.1

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24206)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
👀
image

**Comment by quhxl** [comment]
I know it is not matching, but it is the same software developer as the other package. Should we really have multiple vendor folders if they come from the same vendor?

**Comment by OfficialEsco** [comment]
Ooops didn't see the msgs before now, and yes i was very much referring to the version which took you a bit longer than expected to notice 😁

**Review by OfficialEsco — DISMISSED** [review]
You will probably be told by a technician to test if its works on a x86 machine..

**Comment by quhxl** [comment]
I will change this to x64 because I’m not home rn to test on my x86 machine

File: manifests/s/StefanKueng/StExBar/1.11.1/StefanKueng.StExBar.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  InstallerUrl: https://github.com/stefankueng/tools/releases/download/StExBar_1.11.2/StExBar64-1.11.2.msi
+  InstallerSha256: 3505069A4430A4193FFBAEC2D75EBCB6A4105EB1ACE8BC46AE8BC8DF56ACA437
+  UpgradeBehavior: install
+  ProductCode: '{ACD1B392-D2B0-47BE-8454-1FCFE6B93EEF}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/stefankueng/tools/releases/download/StExBar_1.11.2/StExBar-1.11.2.msi
+  InstallerSha256: 104B763B4BB7AF4807CABA932A621F207FF32F18ECFD1A936C4944F1410FB5BE   UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/s/StefanKueng/StExBar/1.11.1/StefanKueng.StExBar.installer.yaml:25]: Sick revert
```suggestion
  InstallerSha256: 104B763B4BB7AF4807CABA932A621F207FF32F18ECFD1A936C4944F1410FB5BE
  UpgradeBehavior: install
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/s/StefanKueng/StExBar/1.11.1/StefanKueng.StExBar.installer.yaml (hunk 1 above)
Comment (quhxl) [reply manifests/s/StefanKueng/StExBar/1.11.1/StefanKueng.StExBar.installer.yaml]: Just on my phone rn ^^

**Comment by quhxl** [comment]
Since #25065 has been merged, I'm closing this.

**Pull request closed without merging**